m2m-transfers/m2m-transfers-json-contract.md

```diff
@@ -0,0 +1,196 @@
+			"memberLimitCount": "3",
+			"memberLimitAmount": "5000.00",
+			"memberActualCount": "1",
+			"memberActualAmount": "500.00"
+		}],
+		"availableShares": [{
+				"transferSLId": "0000800005S0002",
+				"transferName": "SUMMER SAVER",
+				"available": "50.00"
+			},
+			{
+				"transferSLId": "0000800005S0010",
+				"transferName": "REGULAR CHECKING",
+				"available": "500.00"
+			},
+			{
+				"transferSLId": "0000800005S0015",
+				"transferName": "ULTIMATE CHECKING (10)",
+				"available": "0.00"
+			}
+		],
+
+		"scheduledTransfers": [{
+				"trnsferLoc": "215",
+				"sourceAccount": "1234567890S0020",
+				"accountName": "CHECKING",
+				"transferAmt": "120.00",
+				"recipientName": "Emily Fitch",
+				"recipientMemberId": "9876543210",
+				"recipientAccount": "[9876543210L0001,savings,checking]",
+				"recipientNickname": "Emmy",
+				"startDate":"07/31/21",
+				"endDate":"12/31/2021",
+				"transferFrequency": "weekly",
+				"day1": "",
+				"day2": "",
+				"saveRecipient": false,
+				"memo": "Internal Note created by member"
+			},
+			{
+				"transferLoc": "395",
+				"sourceAccount": "1234567890S0001",
+				"accountName": "SAVINGS",
+				"transferAmt": "120.00",
+				"recipientName": "Howard Cleo",
+				"recipientMemberId": "9876543210",
+				"recipientShareId": "9876543210L0001",
+				"recipientNickname": "Howie",
+				"nextTransferDate": "12/15/2020",
+				"startDate":"07/03/21",
+				"endDate":"12/31/2025",
+				"transferFrequency": "semi-monthly",
+				"day1": "3",
+				"day2": "31",
+				"saveRecipient": false,
+				"memo": "Internal Note created by member"
+			}
+		],
+		"savedRedipients": [{
+				"recipientLoc": "5443231543",
+				"recipientName": "Binsy Flomor",
+				"recipientSLId": "9876543210L0001",
+				"recipientNickName": "Zeke's Future"
+			},
+			{
+				"recipientLoc": "5431543",
+				"recipientName": "Kylie Crenshaw",
+				"recipientSLId": "9876543210L0001",
+				"recipientNickName": "Sally Martin"
+			}
+		]
+	}
+}
+```
+### PowerOn Error Responses (PRELOADDATA)
+#### PowerOn response - Configuration File read error:
+```json
+{
+  "errorCode": "501",
+  "loggingErrorMessage": "Error [opening/reading] from config file: [error msg]"
+}
+```
+#### PowerOn response - Configuration File read error:
+```json
+{
+  "errorCode": "501",
+  "loggingErrorMessage": "Error [opening/reading] from config file: [error msg]"
+}
+```
+### PowerOn response - Invalid Account (by Account Warning):
+
+```json
+{
+  "errorCode": "502",
+  "loggingErrorMessage": "Invalid Account - account warning xxx"
+}
+```
+### PowerOn response - No Eligible Share(s) found:
+
+```json
+{
+  "errorCode": "503",
+  "loggingErrorMessage": "No eligible shares found"
+}
+```
+## PROCESS REQUEST STATE
+### Client Request (PROCESS)
+
+```json
+{
+  "rgState": "PROCESS",
+  "powerOnFileName": "BANNO.M2MTRANSFERS.V1.POW",
+  "userChrList": [
+    { "id": 1, "value": "[request type]:[data]|[data]|[data]|[data]|[data]|[data]" },
```

Comment:
Any reason to not use separate rgState's for the different operations below? I think that would be less confusing than the request type parameter

Reply:
I think it was a "6 of one, half dozen of the other" kind of conversation and that's which side the coin came up on.  I can go with the different state names

Reply:
I think the names you created for the request type could be used for the rgState just as they are.

Reply:
duh!  OK.. definitely a Monday!